assets/pages/bundles-landing/helpers/subscriptionsLinks.js

```diff
@@ -59,9 +59,14 @@ function buildUrls(promoCodes: PromoCodes, intCmp: string): SubsUrls {
 
 }
 
-
 // ----- Exports ----- //
 
+export function belongToTest(intCmp: string, nameOfTest: string): boolean {
+  const test = tests.find(t => t.name === nameOfTest);
+  return test !== undefined && test.intCmps.includes(intCmp);
+
+}
```

Comment:
As discussed offline, this should probably be in a different module.

Reply:
Card created [here](https://trello.com/c/O7KgTKLW/867-refactor-create-a-test-helper-in-support-frontend-where-we-can-centrelize-all-the-tests-and-put-the-belongtotest-function)